{"uuid":"97afc433-27a5-4149-9cbb-6a2457b9df2b","historyId":"80cada435a764b796b0c308b6c8d9479","fullName":"tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword","labels":[{"name":"package","value":"tests.LoginTests"},{"name":"testClass","value":"tests.LoginTests"},{"name":"testMethod","value":"invalidLoginTest_EmptyUserEmptyPassword"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"allurereportexample"},{"name":"subSuite","value":"tests.LoginTests"},{"name":"host","value":"Zoomdatas-MacBook-Pro-3.local"},{"name":"thread","value":"19946@Zoomdatas-MacBook-Pro-3.local.main(1)"},{"name":"epic","value":"Regression Tests"},{"name":"feature","value":"Login Tests"},{"name":"story","value":"Empty username and password login test"},{"name":"severity","value":"critical"}],"links":[],"name":"Invalid Login Scenario with empty username and password.","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [WRONG MESSAGE FOR FAILURE!] but found [Bu alanın doldurulması zorunludur.]","trace":"java.lang.AssertionError: expected [WRONG MESSAGE FOR FAILURE!] but found [Bu alanın doldurulması zorunludur.]\n\tat org.testng.Assert.fail(Assert.java:96)\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:137)\n\tat org.testng.Assert.assertEquals(Assert.java:118)\n\tat org.testng.Assert.assertEquals(Assert.java:453)\n\tat org.testng.Assert.assertEquals(Assert.java:463)\n\tat pages.LoginPage.verifyLoginPassword_aroundBody4(LoginPage.java:43)\n\tat pages.LoginPage$AjcClosure5.run(LoginPage.java:1)\n\tat org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)\n\tat io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)\n\tat pages.LoginPage.verifyLoginPassword(LoginPage.java:43)\n\tat tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword(LoginTests.java:82)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)\n\tat org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)\n"},"stage":"finished","description":"Test Description: Login test with empty username and empty password.","start":1548886005179,"stop":1548886009037,"steps":[{"name":"Open N11 Step...","status":"passed","stage":"finished","start":1548886005180,"stop":1548886006906},{"name":"Go to Login Page Step...","status":"passed","stage":"finished","start":1548886006906,"stop":1548886007531},{"name":"Login Step with username: {0}, password: {1}, for method: {method} step...","status":"passed","stage":"finished","start":1548886007531,"stop":1548886007669,"parameters":[{"name":"username","value":""},{"name":"password","value":""}]},{"name":"Verify username: {0} step...","status":"passed","stage":"finished","start":1548886008171,"stop":1548886008200,"parameters":[{"name":"expectedText","value":"Lütfen e-posta adresinizi girin."}]},{"name":"Verify verifyLoginPassword: {0} step...","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [WRONG MESSAGE FOR FAILURE!] but found [Bu alanın doldurulması zorunludur.]","trace":"java.lang.AssertionError: expected [WRONG MESSAGE FOR FAILURE!] but found [Bu alanın doldurulması zorunludur.]\n\tat org.testng.Assert.fail(Assert.java:96)\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:137)\n\tat org.testng.Assert.assertEquals(Assert.java:118)\n\tat org.testng.Assert.assertEquals(Assert.java:453)\n\tat org.testng.Assert.assertEquals(Assert.java:463)\n\tat pages.LoginPage.verifyLoginPassword_aroundBody4(LoginPage.java:43)\n\tat pages.LoginPage$AjcClosure5.run(LoginPage.java:1)\n\tat org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)\n\tat io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)\n\tat pages.LoginPage.verifyLoginPassword(LoginPage.java:43)\n\tat tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword(LoginTests.java:82)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)\n\tat org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)\n"},"stage":"finished","start":1548886008200,"stop":1548886008226,"parameters":[{"name":"expectedText","value":"WRONG MESSAGE FOR FAILURE!"}]}],"attachments":[{"name":"Page screenshot","source":"9cd6b33e-5025-420b-849c-3608dd6ba003-attachment","type":"image/png"},{"name":"{0}","source":"6616b199-d5a8-47ae-a7fc-d69062ec58ce-attachment","type":"text/plain"}],"parameters":[]}